src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -135,6 +136,9 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
                 new ProgressStepDefinition(null, StepAttributes.Hidden | StepAttributes.Indeterminate,
                         (token, notifications) => this.FinishSolutionBindingOnUIThread(controller, token)),
 
+                new ProgressStepDefinition(Strings.BindingProjectsDisplayMessage, StepAttributes.BackgroundThread | StepAttributes.Indeterminate,
+                        (token, notifications) => this.SaveServerExclusionsAsync(controller, notifications, token).GetAwaiter().GetResult()),
```

Comment:
should we block this?

Reply:
if we don't we can't fail the workflow process.

Reply:
But other steps, like `PrepareSolutionBinding` don't do this. 
Inside SaveServerExclusionsAsync we have a call to AbortWorkflow, so it should be able to handle exceptions.

What would happen if you were to remove this?